numcodecs/blosc.pyx

```diff
@@ -525,9 +525,9 @@ class Blosc(Codec):
     clevel : integer, optional
         An integer between 0 and 9 specifying the compression level.
     shuffle : integer, optional
-        Either NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). If -1
-        (default), bit-shuffle will be used for buffers with itemsize 1,
-        and byte-shuffle will be used otherwise.
+        Either NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). If -1,
```

Comment:
Maybe worth saying `AUTOSHUFFLE` instead of `-1` for clarity

```suggestion
        Either NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). If AUTOSHUFFLE,
```